javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,24 @@
+// * My free code camp link : https://www.freecodecamp.org/fcc733a70bc-2aaa-4346-b4d1-7b85038190f9
+
+// Flight booking fullname function
+
+function getFullname(firstname, surname, sex, useFormalName = false) {
```

Comment:
Only fix I can see is remember to use camelCase (even though I know for this assignment it is spelt incorrectly)